packages/rum-core/src/boot/startRum.ts

```diff
@@ -72,6 +74,7 @@ export function startRum(
     getCommonContext,
     reportError
   )
+  addTelemetryConfiguration(serializeRumConfiguration(initConfiguration))
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏what about colocating this code with other code related to telemetry?

Reply:
As we discussed, colocating the code is not easy due to the telemetry dependencies. I made an e2e test to ensure the feature works. https://github.com/DataDog/browser-sdk/pull/1760/files#r991413769